solution/backend/regulations/admin.py

```diff
@@ -50,10 +60,9 @@
 
     def import_conversions(self, text, url, act):
         conversions = []
-        text = re.sub(r"</?[^>]+>", "", text)  # Strips HTML/XML tags from the response text
-        text = re.sub(r"[—–]", "-", text)  # Replace em dash and en dash with regular dash
-        matches = re.findall(r"[Ss][Ee][Cc].?\s*(\d+[A-Za-z0-9]*(?:-+[A-Za-z0-9]+)?).?\s*\[\s*(\d+)\s*[Uu].?[Ss].?[Cc].?\s*"
-                             r"(\d+[A-Za-z0-9]*(?:-+[A-Za-z0-9]+)?)\s*\]", text)
+        text = markup_regex.sub("", text)  # Strips HTML/XML tags from the response text
```

Comment:
This is fine as it's needed to extract data from the XML. This is user provided, but only by admins and SMEs and therefore has no effect on the end-user experience. If someone tries to enter a URL that deliberately slows things down, it would only time out for that person.